hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -355,6 +355,9 @@ public boolean scheduleExists(ScheduleID id) {
 						Thread.currentThread().interrupt();
 					}
 				}
```

Comment:
There should be an `else` here. We should `log.warn` if we exhausted all the attempts.

Reply:
nvm, I see that we `return Optional.empty();`